config/post_updates/10005/core.entity_form_display.paragraph.oe_fact.default.yml

```diff
@@ -0,0 +1,47 @@
+langcode: en
```

Comment:
This file seems to be different from what stored in config/install, see diff below:

```diff
<     weight: 0
---
>     weight: 1
22c22
<     weight: 3
---
>     weight: 2
30c30
<     weight: 2
---
>     weight: 3
38c38
<     weight: 1
---
>     weight: 0
```

Reply:
IN fact the fields order seem messed up when upgrading from master to this branch:

![image](https://user-images.githubusercontent.com/153362/76749165-5c473e80-677c-11ea-9b79-df55535260b9.png)

Reply:
This is what I get from a clean installation:

![image](https://user-images.githubusercontent.com/153362/76749840-79c8d800-677d-11ea-8c07-7e8db5c4647e.png)